docs/concepts/event-reactor.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Event Reactor
+headline: Event Reactor
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Event Reactor</h2> 
+Event Reactor is an object which usually produces one or more events in response to an incoming event. 
+Unlike <a href="https://spine.io/docs/concepts/event-subsriber.html">Event Subscriber</a>, which always consumes events, a reacting object generates events in response to changes in the domain.
```

Comment:
Not “always”. In some cases, a Reactor may ignore the event, returning `Nothing`. It usually happens when a tuple is declared in the return type of a method, and `Nothing` is one of the possible options.

Reply:
For example, there could be a return type declared like `EitherOf2<TaskReAssigned, Nothing>`.